model.py

```diff
@@ -5482,14 +5490,25 @@ def update_availability(
                 old_value=old_value, new_value=new_value)
 
         # Update the license pool with the latest information.
-        self.licenses_owned = new_licenses_owned
-        self.licenses_available = new_licenses_available
-        self.licenses_reserved = new_licenses_reserved
-        self.patrons_in_hold_queue = new_patrons_in_hold_queue
-        self.last_checked = as_of
+        any_data = False
+        if new_licenses_owned is not None:
+            self.licenses_owned = new_licenses_owned
+            any_data = True
```

Comment:
Unfortunately setting these values to 0 upon creation of the LicensePool was not sufficient. If you have a CirculationData object that has format information but no licensing information, calling update_availability() was replacing the zero values with null values. This puts a stop to that. If the CirculationData doesn't know something, we leave the preexisting value in place.